src/iuse.cpp

```diff
@@ -3603,6 +3604,11 @@ const std::string &get_music_description()
 
         }
     };
+    static const std::array<std::string, 3> genre_snippet_categories = {{
```

Comment:
This array is quite useless. You access each element exactly once and with a fixed index, not using any loop or similar. You can use the string literals directly in the function calls below. And why is this defined so far away from its actual usage?